incubator/vault/templates/deployment.yaml

```
@@ -129,6 +132,9 @@ spec:
             scheme: {{ if .Values.vault.config.listener.tcp.tls_disable -}}HTTP{{- else -}}HTTPS{{- end }}
           initialDelaySeconds: {{ .Values.vault.readiness.initialDelaySeconds }}
           periodSeconds: {{ .Values.vault.readiness.periodSeconds }}
+          failureThreshold: {{ .Values.vault.readiness.failureThreshold }}
+          successThreshold: {{ .Values.vault.readiness.successThreshold }}
+          timeoutSeconds: {{ .Values.vault.readiness.timeoutSeconds }}
```

How about making these values the default unless overridden?

```suggestion
          failureThreshold: {{ .Values.vault.readiness.failureThreshold | default 3 }}
          successThreshold: {{ .Values.vault.readiness.successThreshold | default 1 }}
          timeoutSeconds: {{ .Values.vault.readiness.timeoutSeconds | default 1 }}
```

This way we only have to set these in the values.yaml when needed?

---

I realize that `vault.readiness.initialDelaySeconds` and vault.readiness.periodSeconds` should also follow the same convention and currently they don't! 😄 

Same goes for the `liveness` thresholds!